### PR TITLE
VUE-693 design microsite scrollspy fixes

### PR DESCRIPTION
**Pull request by ryan-ludwig**

### CODE REVIEW

File: src/pages/ContentfulPage.vue [hunk 1]
```diff
@@ -12,6 +12,7 @@
 				:content="content"
 				v-bind="getComponentOptions(content.key)"
 				:class="wrapperClass"
+				data-section-type="contentful-section"
```
Comment (ryan-ludwig) [inline src/pages/ContentfulPage.vue:15]: Open to other names/selectors here, just needed a way to queryselector all of the contentful sections.

**Comment by ryan-ludwig** [comment]
Fixes/improves upon the navigation scrollspy. Unsure how to fix the timeout problem, but need to move off this bug soon.

**Comment by ryan-ludwig** [comment]
https://user-images.githubusercontent.com/187937/135352785-d43a2b99-9767-42f9-ae17-05cd2484e3f7.mov

**Review by mcstover — APPROVED** [review]
Quite a lot to highlight a nav item ;)

**Review by emuvente — APPROVED** [review]

**Pull request merged**